src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,68 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: CallableFunction,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const lastPageId = Math.ceil(total / perPage);
+  const handlePageChange = (step:number) => {
+    onPageChange((prevPage: number) => prevPage + step);
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled="true"
```

Comment:
1. `«` and `»` links should open the prev and the next pages accordingly
    - disable each of them if it is already the first or the last page (use `li.disabled` and `a[aria-disabled="true"]`)